content/codes/content.md

```diff
@@ -969,8 +969,50 @@ https://www.youtube.com/watch?v=mcX7iO_XCFA
 
 {.todo} TODO
 
+    div.rotors#rotors
+      div.rotorctl
+        span#rotor3up &#8593;
```

Comment:
We should never use IDs like `#rotor3up`, so that we can reuse the same component multiple times on a page. Let's use classes instead. There are quite a lot of other elements with IDs below...

(FYI, `div` is the default element type in PUG, so you don't need to specify that explicitly.